src/components/AddBook.js

```diff
@@ -1,11 +1,45 @@
-import React from 'react';
-
-const AddBook = () => (
-  <form>
-    <input type="text" placeholder="title" />
-    <input type="text" placeholder="author" />
-    <button type="submit">Add Book</button>
-  </form>
-);
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import uuid from 'react-uuid';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const [newBook, setNewBook] = useState(
+    {
+      id: '',
+      title: '',
+      author: '',
+    },
+  );
+
+  const handleSubmitBook = (e) => {
+    e.preventDefault();
+    setNewBook({ ...newBook, id: uuid });
+    dispatch(addBook(newBook));
+  };
+
+  return (
+    <form onSubmit={handleSubmitBook}>
+      <h3>ADD NEW BOOK</h3>
+      <div className="form-control">
+        <input
+          placeholder="Book title"
+          value={newBook.title}
+          onChange={(e) => setNewBook({ ...newBook, title: e.target.value })}
+        />
+        <input
+          placeholder="Author"
+          value={newBook.author}
+          onChange={(e) => setNewBook({ ...newBook, author: e.target.value })}
+        />
```

Comment:
- [ OPTIONAL ] I recomend make the inputs empty because it is very important for user experience so kindly after submitting new book make the inputs empty again 🤗✨🙏🏻

Reply:
Thank you for the feedback. I will change the input as an empty after form submission.